src/perf/lib/quicmain.cpp

```diff
@@ -69,7 +69,7 @@ QuicMainStart(
         return QUIC_STATUS_INVALID_PARAMETER;
     }
 
-    const char* TestName = GetValue(argc, argv, "test");
+    const char* TestName = GetValue(argc, argv, "TestName");
```

Comment:
I don't mind having `test` to be an alias to `TestName` here, but we need to support both. For instance, if `TestName` isn't present, then check for `test`.

Reply:
> I don't mind having `test` to be an alias to `TestName` here, but we need to support both. For instance, if `TestName` isn't present, then check for `test`.

That would be better:
a simple short circuit may do?
```
const char* TestName = GetValue(argc, argv, "TestName") || GetValue(argc, argv, "test");
```

Reply:
Essentially, but I think you'll need separate statements.

Reply:
@nibanks Please check out 00bd717, linked below.